packages/manager/modules/pci/src/projects/project/storages/object-storage/users/import/import.module.js

```diff
@@ -0,0 +1,32 @@
+import angular from 'angular';
+import '@ovh-ux/ng-translate-async-loader';
+import '@uirouter/angularjs';
+import 'angular-translate';
+import '@ovh-ux/ui-kit';
+import 'ovh-api-services';
+
+import component from './import.component';
+import routing from './import.routing';
+
+const moduleName = 'ovhManagerPciStoragesObjectsUserListImport';
+
+angular
+  .module(moduleName, [
+    'ui.router',
+    'oui',
+    'ovh-api-services',
+    'ngTranslateAsyncLoader',
+    'pascalprecht.translate',
+  ])
+  .config(routing)
+  .component('pciStoragesObjectsUserListImport', component)
+  .config(
+    /* @ngInject */ ($compileProvider) => {
+      $compileProvider.aHrefSanitizationWhitelist(
```

Comment:
According to documentation: https://docs.angularjs.org/api/ng/provider/$compileProvider#aHrefSanitizationWhitelist deprecated since 1.18.1 

Also wouldn't that be applied to all other places once module has been loaded once ?

Reply:
Have removed the deprecated code here. But this is also being used in 2 other places. I believe we must add some file url `sanitization` in `<oui-file>` module itself